app/models/reservation.rb

```diff
@@ -6,4 +6,19 @@ class Reservation < ApplicationRecord
   has_many :tickets, dependent: :destroy
 
   enum :status, %i[canceled booked confirmed]
+
+  scope :relevant_for, lambda { |user|
+    joins(:user, :show)
+    .where(user:)
+    .where(show: { start_time: Time.zone.now.. })
+    .where.not(status: :canceled)
+    .order(:start_time)
+  }
```

Comment:
Sure, gonna change that. It's strange that Rubocop didn't say anything about that.

Reply:
I looked into that and simply replacing one-line blocks with multiline blocks is not possible here. It's due to the fact that different blocks have different precedence and when you use `do ... end`, interpreter will think that they belong to scope function instead of lambda.

In case someone has a similar issue, more details are available [here](https://github.com/rubocop/rubocop/issues/1520).